gradle/libs.versions.toml

```diff
@@ -1,14 +1,17 @@
 [versions]
+# javalite, protoc and protobuffjavautil versions should be in sync while updating and
+# it needs to match the protobuff version which grpc has transitive dependency on.
 android-lint = "30.3.1"
 autovalue = "1.10.1"
 coroutines = "1.6.4"
 dagger = "2.43.2"
-grpc = "1.50.2"
-javalite = "3.17.3"
+grpc = "1.52.1"
+javalite = "3.21.7"
 kotlin = "1.7.10"
-protoc = "3.17.3"
+protoc = "3.21.7"
 robolectric = "4.9"
 truth = "1.1.2"
+protobuffjavautil = "3.21.7"
```

Comment:
For completeness, please also add this library to the list under the testing "section".

Reply:
Not sure i understand where to put.

Reply:
I think what @rlazo meant was to add this library under the `# Test Libs` line, since it's only used during testing:

https://github.com/firebase/firebase-android-sdk/blob/1e3228eba7f91607b00a2c0aa0d8267f3fda3b6e/gradle/libs.versions.toml#L40-L50

So it would be something like:

```toml
protobuf-java-util = { module = "com.google.protobuf:protobuf-java-util", version = "3.21.7" }
```

Reply:
Added now, please check. Thanks